### DISCUSSION TITLE
API URL

### DISCUSSION THREAD

**Discussion by aquaosrs** [TomBursch/kitchenowl#280] [Q&A]
Hey,

I'm having some trouble connecting to the API to list the shopping.

I've read this:

https://github.com/TomBursch/kitchenowl/discussions/200

Which is great but I'm not sure what address to run this against.

I have my frontend running at localhost:9090 (using 9090:80 against the front container)

I have my backend running also using the standard docker-compose.

I have tried exposing my backend server in both:

9192:5000
and
9191:80

but http get calls via postman are returning Error: connect ECONNREFUSED with nothing in the logs of the Docker backend service.

Could you help me with either which ports to expose and on which container, or what route I could use on the frontend to get ```/api/v1/shoppinglist``` to return something

**Comment by TomBursch** [comment]
Hey, thanks for being patient. The API is located at `localhost:9090/api`. I'm not sure why I said it wrong in the other comment, but `v1` is not part of the API path (I think I wanted to migrate to `v1` but didn't).

**Reply by aquaosrs** [reply]
Perfect thanks, to anyone else reading this the most simple query you can do it on your frontend is a get on this:

```locahost:9090/api/household/1/shoppinglist``` 
you have to first specify the household ID
